packages/flutter/test/cupertino/picker_test.dart

```diff
@@ -343,14 +343,19 @@ void main() {
       );
 
       // Drag it by a bit but not enough to move to the next item.
-      await tester.drag(find.text('10'), const Offset(0.0, 30.0), pointer: 1, touchSlopY: 0.0, warnIfMissed: false); // has an IgnorePointer
+      TestGesture gesture = await tester.startGesture(tester.getCenter(find.text('10')));
 
-      // The item that was in the center now moved a bit.
+      // Use a custom gesture to work around the DragStartBehavior.start.
+      // First 20 initiates the drag, next 30 is the actual drag.
+      await gesture.moveBy(const Offset(0.0, 20.0));
+      await gesture.moveBy(const Offset(0.0, 30.0));
```

Comment:
I'm a little confused by this. What change makes this needed for there to be a stutter gesture before the actual gesture? Does this mean the picker is a little less responsive initially? Where was `DragStartBehavior.start` set compared to before?

Reply:
Previously, the scroll behavior defined in provided scroll controllers conflicted with this default tap-to-scroll behavior. Since we now make provided scroll controllers customize their own tap-to-scroll behavior, those changes aren't needed anymore.